h/static/scripts/directive/test/annotation-test.js

```diff
@@ -149,196 +149,116 @@ describe('annotation.js', function() {
     });
   });
 
-  describe('updateViewModel()', function() {
-    var updateViewModel = require('../annotation').updateViewModel;
-    var sandbox;
-
-    beforeEach(function() {
-      sandbox = sinon.sandbox.create();
-    });
-
-    afterEach(function() {
-      sandbox.restore();
-    });
+  describe('updateDomainModel()', function() {
+    var updateDomainModel = require('../annotation').updateDomainModel;
 
-    /** Return a mock of the `drafts` service. */
-    function mockDrafts() {
+    function fakePermissions() {
       return {
-        get: function() {}
+        shared: function() {},
+        private: function() {},
       };
     }
 
-    it('copies model.document.title to vm.document.title', function() {
-      var vm = {};
-      var model = {
-        uri: 'http://example.com/example.html',
-        document: {
-          title: 'A special document'
-        }
+    function fakeGroups() {
+      return {
+        focused: function() {return {};},
       };
+    }
 
-      updateViewModel(mockDrafts(), model, vm);
-
-      assert.equal(vm.document.title, 'A special document');
-    });
-
-    it('uses the first title when there are more than one', function() {
-      var vm = {};
-      var model = {
-        uri: 'http://example.com/example.html',
-        document: {
-          title: ['first title', 'second title']
-        }
-      };
+    it('copies text from viewModel into domainModel', function() {
```

Comment:
I'm not asking for any further changes here for the moment, but if I were new to this code I think I'd find some of this quite difficult to understand, because it is very much expressed in terms of the language of the implementation, rather than the API of the component.